sections/azera_shop_ribbon_section.php

```diff
@@ -16,6 +16,9 @@
 	$azera_shop_ribbon_title = get_theme_mod( 'azera_shop_ribbon_title' );
 	$azera_shop_button_text = get_theme_mod( 'azera_shop_button_text' );
 }
+$ribbon_background = apply_filters( 'azera_shop_translate_single_string', $ribbon_background, 'Ribbon section - background image' );
+$azera_shop_ribbon_title = apply_filters( 'azera_shop_translate_single_string', $azera_shop_ribbon_title, 'Ribbon section - title' );
```

Comment:
Here, I know you wanted those descriptions to be more clear like "Ribbon section - title", but for wporg themes, we want to keep the number of new strings as lower, to keep the translations.

So, I think it may be a good idea to use the same string for each group